tasks/shell_crmsh/check-and-prepare-role-variables.yml

```diff
@@ -0,0 +1,127 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check cluster configuration variables
+  block:
+    - name: Fail if passwords are not specified
+      ansible.builtin.fail:
+        msg: "{{ item }} must be specified"
+      when:
+        - lookup("vars", item, default="") | string | length < 1
+        - ha_cluster_cluster_present | bool
+      loop:
+        - ha_cluster_hacluster_password
+      run_once: true
```

Comment:
If there is any chance this task will display the cleartext password when using a verbose log level (e.g. `ansible-playbook -vv ...`), then you should add `no_log: true` to this task

Reply:
I will add it, but pcs file also does not have it since it is one of tasks that was copied  ... do you want me to add it to pcs as well?

https://github.com/linux-system-roles/ha_cluster/blob/fcd3461ec792a9b5de3a01a6f423d8d5a6942833/tasks/shell_pcs/check-and-prepare-role-variables.yml#L15C1-L25C21

Reply:
This doesn't print the password even with `-vv` for me...

Reply:
thank you for checking, I will keep it in crmsh just in case.